coverage-report.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+CONFIGS="Release Debug"
+COVERAGE_PROFILE_DIRS="
+  common/cpp/build/tests
+  third_party/libusb/webport/build/tests"
+EXECUTABLE_MODULE_DIRS="
+  smart_card_connector_app/build/executable_module"
+UNIT_TEST_EXECUTABLE_DIRS="
+  common/cpp/build/tests
+  third_party/libusb/webport/build/tests"
+IGNORE_FILENAME_REGEX="
+  third_party/googletest"
```

Comment:
BTW, should we also exclude the unit test files? ("*test.cc")
Not sure what's the common practice for calculating the overall coverage percentage.

Reply:
Unit tests provide coverage as well. 
IMHO we could start with having them and we will see if some lines are too exhaustively covered. Then, if needed, we will take it from there.